rosidl_runtime_py/convert.py

```diff
@@ -196,52 +196,54 @@ def _convert_value(
     value,
     *,
     field_type=None,
-    truncate_length=None,
+    truncate_len=None,
```

Comment:
Please keep this argument name as `truncate_length`. This appears to be the cause of some of the test failures.